FTBFS on AOSC OS

Cockpit version: 188
OS: 
```
root@saki-exp-40001 [ cockpit-188 ] ! uname -a && gcc --version && lsb_release -a
Linux saki-exp-40001 4.19.19-aosc-main #1 SMP PREEMPT Mon Feb 4 11:22:09 CST 2019 x86_64 GNU/Linux                                                                     
gcc (GCC) 7.3.1 20180414 (AOSC OS, Core)
Copyright (C) 2017 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

LSB Version:    n/a
Distributor ID: AOSC OS
Description:    AOSC OS
Release:        5.3.2
Codename:       eMMC
```
<!--- Issue description -->

```
root@saki-exp-40001 [ build ] # make
make  all-am
make[1]: Entering directory '/var/cache/acbs/build/acbs.5hapn63f/cockpit-188/build'
  CCLD     cockpit-ssh
/bin/ld: error: src/ssh/cockpit_ssh-ssh.o: requires dynamic R_X86_64_PC32 reloc against 'dup' which may overflow at runtime; recompile with -fPIC
/bin/ld: error: libcockpit-ssh.a(libcockpit_ssh_a-cockpitsshrelay.o): requires dynamic R_X86_64_32 reloc which may overflow at runtime; recompile with -fPIC
/bin/ld: error: libcockpit-ssh.a(libcockpit_ssh_a-cockpitauthorize.o): requires dynamic R_X86_64_PC32 reloc against 'strchr' which may overflow at runtime; recompile with -fPIC
/bin/ld: error: libcockpit-ssh.a(libcockpit_ssh_a-cockpitsshoptions.o): requires dynamic R_X86_64_PC32 reloc against 'g_environ_getenv' which may overflow at runtime; recompile with -fPIC
/bin/ld: error: libcockpit-common.a(libcockpit_common_a-cockpitbase64.o): requires dynamic R_X86_64_PC32 reloc against '__ctype_b_loc' which may overflow at runtime; recompile with -fPIC
/bin/ld: error: libcockpit-common.a(libcockpit_common_a-cockpitframe.o): requires dynamic R_X86_64_32 reloc which may overflow at runtime; recompile with -fPIC
/bin/ld: error: libcockpit-common.a(libcockpit_common_a-cockpitjson.o): requires dynamic R_X86_64_PC32 reloc against 'strlen' which may overflow at runtime; recompile with -fPIC
/bin/ld: error: libcockpit-common.a(libcockpit_common_a-cockpitlog.o): requires dynamic R_X86_64_32 reloc which may overflow at runtime; recompile with -fPIC
/bin/ld: error: libcockpit-common.a(libcockpit_common_a-cockpitmemory.o): requires dynamic R_X86_64_PC32 reloc against 'memset' which may overflow at runtime; recompile with -fPIC
/bin/ld: error: libcockpit-common.a(libcockpit_common_a-cockpitpipe.o): requires unsupported dynamic reloc 11; recompile with -fPIC
/bin/ld: error: libcockpit-common.a(libcockpit_common_a-cockpittest.o): requires dynamic R_X86_64_PC32 reloc against 'g_free' which may overflow at runtime; recompile with -fPIC
/bin/ld: error: libcockpit-common.a(libcockpit_common_a-cockpittransport.o): requires dynamic R_X86_64_32 reloc which may overflow at runtime; recompile with -fPIC
/bin/ld: error: libcockpit-common.a(libcockpit_common_a-cockpitunixfd.o): requires dynamic R_X86_64_PC32 reloc against 'close' which may overflow at runtime; recompile with -fPIC
/bin/ld: error: libcockpit-common.a(libcockpit_common_a-cockpitconf.o): requires dynamic R_X86_64_PC32 reloc against 'g_key_file_new' which may overflow at runtime; recompile with -fPIC
/bin/ld: error: libcockpit-common.a(libcockpit_common_a-cockpitflow.o): requires dynamic R_X86_64_32 reloc which may overflow at runtime; recompile with -fPIC
/bin/ld: error: libcockpit-common.a(libcockpit_common_a-cockpithash.o): requires dynamic R_X86_64_PC32 reloc against 'g_ascii_tolower' which may overflow at runtime; recompile with -fPIC
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:3569: cockpit-ssh] Error 1
make[1]: Leaving directory '/var/cache/acbs/build/acbs.5hapn63f/cockpit-188/build'
make: *** [Makefile:2951: all] Error 2
```
Steps to reproduce

1. mkdir build && cd build
2. ../configure
3. make

--- a/src/common/Makefile-common.am
+++ b/src/common/Makefile-common.am
@@ -98,6 +98,7 @@ libcockpit_common_a_SOURCES = \
 	$(NULL)
 
 libcockpit_common_a_CFLAGS = \
+	-fPIC \
 	-DG_LOG_DOMAIN=\"cockpit-protocol\" \
 	$(COCKPIT_CFLAGS) \
 	$(NULL)

--- a/src/ssh/Makefile-ssh.am
+++ b/src/ssh/Makefile-ssh.am
@@ -20,6 +20,7 @@ libcockpit_ssh_a_SOURCES += \
 endif
 
 libcockpit_ssh_a_CFLAGS = \
+	-fPIC \
 	-I$(top_srcdir)/src/ssh \
 	-DCOCKPIT_BUILD_INFO=\"$(COCKPIT_BUILD_INFO)\"	\
 	-DG_LOG_DOMAIN=\"cockpit-ssh\" \

